src/applications/letters/containers/AddressSection.jsx

```diff
@@ -280,31 +278,14 @@ export class AddressSection extends React.Component {
   render() {
     const address = this.props.savedAddress || {};
     const emptyAddress = isAddressEmpty(this.props.savedAddress);
-    // Street address: first line of address
-    const streetAddressLines = [
-      address.addressOne,
-      address.addressTwo ? `, ${address.addressTwo}` : '',
-      address.addressThree ? ` ${address.addressThree}` : '',
-    ];
-    const streetAddress = streetAddressLines.join('').toLowerCase();
-
-    // City, state, postal code: second line of address
-    const zipCode = getZipCode(address);
-    const city = address.city || '';
-    let cityStatePostal;
-    if (isDomesticAddress(address)) {
-      const state = getStateName(address.stateCode);
-      cityStatePostal = `${city}, ${state} ${zipCode}`;
-    } else if (isMilitaryAddress(address)) {
-      const militaryStateCode = address.stateCode || '';
-      cityStatePostal = `${city}, ${militaryStateCode} ${zipCode}`;
-    } else {
-      // Must be an international address, only show a city
-      cityStatePostal = `${city}`;
-    }
 
-    const country = isInternationalAddress(address) ? address.countryName : '';
-    const addressContentLines = { streetAddress, cityStatePostal, country };
+    const addressContentLines = emptyAddress
+      ? {}
```

Comment:
looking over this, I realized I'm doing sort of double duty and safeguarding against errors both here and in the updated helper functions. I'm leaning towards removing the ternary here and just delegating to the helpers unless folks have a strong preference